server.js

```diff
@@ -138,6 +138,7 @@ var renderChart = (request, window, callback) => {
   var height = request.query.height || 600;
   var lineColors = (request.query.line_colors || '').split(',');
   var legendLabels = (_.compact((request.query.legend_labels || '').split(',')));
+  var yTickFormatter = tickFormatter(request.query.format);
```

Comment:
would rather you do `request.query.format || 'something'` and then make `tickFormatter` either a map or use a switch